BenKimball11/Kennels
# finished through chapter 8 adding animals, locations and employees

#### Changes Made
1. Added file `AnimalForm, EmployeeForm, LocationForm` to `Kennels` directory.
1. Modified file `AnimalForm, EmployeeForm, LocationForm, ApplicationViews` to include `adding new animals, employees, and locations to database` and functionality.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout bk-addingAnimals
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > Instructions for how reviewer can test functionality, and detailed description of what the expected outcome is.
    > Example: When user does BLANK, then BLANK should happen.
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
